src/ifc-reader/operators.cxx

```diff
@@ -218,7 +218,7 @@ namespace ifc::util {
         template<typename Operator>
         using OperatorNameTable = std::map<Operator, const char*>;
 
-        template<typename Op, int N>
+        template<typename Op, std::size_t N>
```

Comment:
Let's use `auto` instead of `std::size_t`

Reply:
Interestingly, this one does not freak IntelliSense out.

Reply:
> Interestingly, this one does not freak IntelliSense out.

Could I convince you to report the repo as a bug to the VS team on the VS DevCom portal?

Reply:
From your response on the other hunk it looked like this is a known issue, but I can indeed file a report.

Reply:
> I can indeed file a report

Please do report, it never hurts :-)